Source/WebCore/platform/mediastream/RealtimeMediaSource.cpp

```diff
@@ -510,7 +510,7 @@ std::optional<MediaConstraintType> RealtimeMediaSource::hasInvalidSizeFrameRateA
         if (std::isinf(constraintDistance)) {
 #if !RELEASE_LOG_DISABLED
             auto range = capabilities.frameRate();
-            ERROR_LOG_IF(m_logger, LOGIDENTIFIER, "RealtimeMediaSource::supportsSizeFrameRateAndZoom failed frame rate constraint, capabilities are [%d, %d]", range.longRange().min, range.longRange().max);
+            ERROR_LOG_IF(m_logger, LOGIDENTIFIER, "RealtimeMediaSource::supportsSizeFrameRateAndZoom failed frame rate constraint, capabilities are [%f, %f]", range.doubleRange().min, range.doubleRange().max);
```

Comment:
As in the other patch, this won't log what it means to. It should be something like this instead:

`capabilities are [", range.doubleRange().min, ", ", range.doubleRange().max, "]"`